src/components/BottomBar/SkyBrowser/SkyBrowserNearestImagesList.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import PropTypes from 'prop-types';
+import CenteredLabel from '../../common/CenteredLabel/CenteredLabel';
+import {FilterList, FilterListData} from '../../common/FilterList/FilterList';
+import styles from './SkyBrowserNearestImagesList.scss';
+import SkyBrowserFocusEntry from './SkyBrowserFocusEntry';
+
+function SkyBrowserNearestImagesList({
+  activeImage,
+  height,
+  selectImage,
+  currentBrowserColor
+}) {
```

Comment:
should the `{` be on a new line?

Reply:
Skipping this for now - I think we need to make a decision (there are many files with this style now)